src/styles/main.scss

```diff
@@ -1,3 +1,16 @@
-body {
+@import "./cards/cards_utils/_variables";
+@import "./header/_variables-header";
+@import "./header/header-block.scss";
+@import "./cards/cards_blocks/card.scss";
+@import "./cards/cards_blocks/stars.scss";
+@import "./main/main-block.scss";
+
+* {
+  box-sizing: border-box;
   margin: 0;
```

Comment:
it's better to put your margin in body tag

* only for box-sizing: border-box;